### PR TITLE
chore: Fix linter findings for `revive:max-public-structs` in `plugins/inputs/[a-n]*`

### PR DESCRIPTION
**Pull request by zak-pawel**
## Summary

Address following findings for [revive:max-public-structs](https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#max-public-structs) in `plugins/inputs/[a-g]*`:
```
plugins/inputs/activemq/activemq.go:2:1                                revive  max-public-structs: you have exceeded the maximum number of public struct declarations
plugins/inputs/ctrlx_datalayer/ctrlx_datalayer_subscription.go:1:1     revive  max-public-structs: you have exceeded the maximum number of public struct declarations
plugins/inputs/dcos/client.go:1:1                                      revive  max-public-structs: you have exceeded the maximum number of public struct declarations
plugins/inputs/graylog/graylog.go:2:1                                  revive  max-public-structs: you have exceeded the maximum number of public struct declarations
plugins/inputs/kafka_consumer/kafka_consumer.go:2:1                    revive  max-public-structs: you have exceeded the maximum number of public struct declarations
plugins/inputs/kubernetes/kubernetes_metrics.go:1:1                    revive  max-public-structs: you have exceeded the maximum number of public struct declarations
plugins/inputs/logstash/logstash.go:2:1                                revive  max-public-structs: you have exceeded the maximum number of public struct declarations
plugins/inputs/mailchimp/chimp_api.go:1:1                              revive  max-public-structs: you have exceeded the maximum number of public struct declarations
plugins/inputs/mongodb/mongostat.go:8:1                                revive  max-public-structs: you have exceeded the maximum number of public struct declarations
plugins/inputs/monit/monit.go:2:1                                      revive  max-public-structs: you have exceeded the maximum number of public struct declarations
```

As proposed here https://github.com/influxdata/telegraf/issues/15822, we will fix the errors found by the following configuration:
```
      - name: max-public-structs
        exclude: ["TEST"]
        arguments: [5]

  # Excluding configuration per-path, per-linter, per-text and per-source
  exclude-rules:
    - path-except: ^plugins/(aggregators|inputs|outputs|parsers|processors|serializers)/...
      text: "max-public-structs: you have exceeded the maximum number of public struct declarations"
```

It is only part of the bigger job.
After all findings of this type in whole project are handled, we can enable `revive:max-public-structs` rule in `golangci-lint`.

## Checklist

- [x] No AI generated code was used in this PR

### CODE REVIEW

File: plugins/inputs/logstash/logstash.go [hunk 1]
```diff
@@ -524,3 +513,16 @@ func init() {
 		return NewLogstash()
 	})
 }
+
+// NewLogstash create an instance of the plugin with default settings
+func NewLogstash() *Logstash {
```
Comment (srebhan) [inline plugins/inputs/logstash/logstash.go:518]: I don't think we need to export this, do we?

**Review by srebhan — COMMENTED** [review]
Thanks @zak-pawel! Just two small comments while we are at it. I also realized that we can probably remove some non-content comments on structs but this is optional...

File: plugins/inputs/logstash/logstash.go [hunk 2]
```diff
@@ -524,3 +513,16 @@ func init() {
...
+	return &Logstash{
+		URL:            "http://127.0.0.1:9600",
+		SinglePipeline: false,
```
[the remark this answers is not in the document]
Comment (zak-pawel) [reply plugins/inputs/logstash/logstash.go]: Removed

**Review by srebhan — APPROVED** [review]
Thanks @zak-pawel!

**Review by DStrand1 — APPROVED** [review]

**Pull request merged**